definitions/ext-snowflake_account/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: EXT
+type: SNOWFLAKE_ACCOUNT
+
+synthesis:
+  identifier: snowflakeAccount
+  name: snowflakeAccount
+
+  conditions:
+    # The attribute’s value must match the provided value
+    - attribute: eventType
+      value: SnowflakeAccount
+  
+compositeMetrics:
+  goldenMetrics:
+    - ./golden_metrics.yml
+  summaryMetrics:
+    - ./summary_metrics.yml
```

Comment:
```suggestion
  goldenMetrics:
    - golden_metrics.yml
  summaryMetrics:
    - summary_metrics.yml
```

Reply:
This was the issue, we removed the relative references, I'll commit this suggestions to verify that everything else is correct.